README.md

```diff
@@ -53,17 +53,17 @@ operations on the wallets. The package has excellent documentation for a smooth
 # Getting Started
 
 - library documentation available at [pkg.go.dev](https://pkg.go.dev/github.com/NethermindEth/starknet.go).
-- [simple call example](./examples/simpleCall) to make a contract call to a mainnet contract
+- [simple call example](./examples/simpleCall) to make a contract call to a testnet contract.
 - [deploy account example](./examples/deployAccount) to deploy a new account contract on testnet.
 - [invoke transaction example](./examples/simpleInvoke) to add a new invoke transaction on testnet.
+- [deploy contract UDC example](./examples/deployContractUDC) to deploy an ERC20 token using [UDC (Universal Deployer Contract)](https://docs.starknet.io/architecture-and-concepts/accounts/universal-deployer/) on testnet.
 
```

Comment:
Let's link this [general examples' FAQ](https://github.com/NethermindEth/starknet.go/tree/main/examples) somewhere in the code (it can be on this line or wherever you think it fits best)

Reply:
Its done.